src/cli/options.rs

```diff
@@ -54,15 +54,15 @@ impl GeneralArgs {
     fn from_matches(matches: &ArgMatches) -> Result<Self, Error> {
         let server = {
             let mut server = match env::var(KRILL_CLI_SERVER_ENV) {
-                Ok(server_str) => Some(uri::Https::try_from(server_str)?),
+                Ok(server_str) => Some(idexchange::ServiceUri::try_from(server_str)?),
```

Comment:
Why `try_from` here and `from_str` in the other uses in this file?

Reply:
no good reason. The old code was doing the same (but then on uri::Https) and this was kept.

That said, I agree that it reads better to just do `from_str` so I modified the code :)